### PR TITLE
EN 1129 user roles page

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/roles/list/RoleListMenuActions.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <LinkMenuItem
+          id={`edit-${this.props.code}`}
+          route={ROUTE_GROUP_EDIT}
+          params={{ groupCode: this.props.code }}
+          label={editLabel}
+          className="RoleListMenuAction__menu-item-edit"
+        /> */}
+        <MenuItem
+          className="RoleListMenuAction__menu-item-delete"
+          onClick={this.handleClick(onClickDelete)}
```
Comment (alessioscalici) [inline src/ui/roles/list/RoleListMenuActions.js]: handleClick is useless if we are using MenuItem (it already ev.preventDefault()), please remove it

File: src/ui/roles/list/RoleListMenuActions.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  render() {
+    const { onClickDelete } = this.props;
+
+    // const editLabel = <FormattedMessage id="app.edit" />;
+    // const editDetails = <FormattedMessage id="app.details" />;
+    return (
+      <DropdownKebab pullRight id={`${this.props.code}-actions`}>
+        {/* <LinkMenuItem
```
Comment (alessioscalici) [inline src/ui/roles/list/RoleListMenuActions.js]: Can you please create the route in "app-init/router", and activate all the links? This way, we'll have all the links in place, pointing to a 404 page, and when we create the related page we just have to update App.js.

File: test/state/roles/actions.test.js [hunk 3]
```diff
@@ -1,18 +1,47 @@
...
+import { TOGGLE_LOADING } from 'state/loading/types';
 import { SET_PAGE } from 'state/pagination/types';
+import { ADD_ERRORS } from 'state/errors/types';
 
 const middlewares = [thunk];
 const mockStore = configureMockStore(middlewares);
 
-config(mockStore({ api: { useMocks: true }, currentUser: { token: 'test_token' } }));
-jest.unmock('api/roles');
+// config(mockStore({ api: { useMocks: true }, currentUser: { token: 'test_token' } }));
```
Comment (alessioscalici) [inline test/state/roles/actions.test.js]: Please remove these comments if they're not useful

File: test/state/roles/actions.test.js [hunk 4]
```diff
@@ -58,47 +88,36 @@ describe('state/roles/actions', () => {
...
-        expect(actionPayload).toHaveProperty('page', 2);
-        expect(actionPayload).toHaveProperty('pageSize', 2);
-        expect(actionPayload).toHaveProperty('lastPage', 5);
-        expect(actionPayload).toHaveProperty('totalItems', 10);
+  describe('sendPostRole()', () => {
+    it('when postRoles succeeds should call post action', (done) => {
+      postRoles.mockReturnValueOnce(new Promise(resolve => resolve(POST_ROLE_PROMISE)));
+      store.dispatch(sendPostRole(BODY_OK)).then(() => {
+        expect(postRoles).toHaveBeenCalled();
+        // expect(gotoRoute).toHaveBeenCalledWith(ROUTE_ROLE_LIST);
```
Comment (alessioscalici) [inline test/state/roles/actions.test.js]: Why commented out?

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**